packages/cozy-harvest-lib/src/components/Datacards/index.jsx

```diff
@@ -26,7 +26,7 @@ const Datacards = ({ konnector, account, trigger }) => {
           konnector={konnector}
           trigger={trigger}
           accountId={trigger.message.account}
-          sourceAccountIdentifier={models.account.getAccountLogin(account)}
+          sourceAccountIdentifier={models.account.getAccountName(account)}
```

Comment:
nit: Do we need a specific version of cozy-client to have this behavior? Is it the right one in dev & peerDeps ?

Reply:
Yes, the 35.1.0 version of cozy-client is needed to have this behavior, we are in 45.8.1 in deps and peer deps.